rackcity/views/asset_views.py

```diff
@@ -1047,7 +1028,7 @@ def asset_bulk_export(request):
 
 
 @api_view(['POST'])
-@permission_classes([IsAdminUser])
+@permission_required(PermissionPath.ASSET_WRITE.value, raise_exception=True)
```

Comment:
might need to check on Piazza whether network bulk actions count as asset actions, but I would think that they do